shor/gates.py

```diff
@@ -13,16 +13,49 @@
 class _Gate(_Layer):
     """Abstract base quantum gate class
 
-    # Properties
     input_length = valid length of input qubits
     qubits = indices of qubits, to be used as input to gate.
+
+    Attributes
+    __________
+    input_length : int
+       valid length of input qubits
+    qubits : int
+       indices of qubits, to be used as input to gate
+
+    Methods
+    -------
+    symbol(self): Returns matrix symbol as lower case for provider transpiler
```

Comment:
nit: ad spaces before Returns for all of these.

find `:Returns` -> `: Returns`